src/gui/editors/PianoRoll.cpp

```diff
@@ -2868,7 +2868,8 @@ void PianoRoll::paintEvent(QPaintEvent * pe )
 	// set font-size to 8
 	p.setFont( pointSize<8>( p.font() ) );
 	QFontMetrics fontMetrics(p.font());
-	int const fontHeight = fontMetrics.boundingRect(QChar::fromLatin1('H')).height();
+	QRect const boundingRect = fontMetrics.boundingRect(QChar::fromLatin1('H'));
+	int const fontHeight = - boundingRect.top() - boundingRect.bottom();
```

Comment:
Now `fontHeight` isn't the height anymore, it's two times of the y coordinate of the center of the bounding rectangle.

Reply:
I can't find any intuitive name, maybe `boundingRectY`, `fontDeltaY` any suggestions?

Reply:
Is that not, per definition, the height? The bounding box bottom should align with the bottom of the text, or it's not a true bounding box. Same for the top. Finally, top - bottom = height, no? It includes ascenders and descenders, but that's not incorrect since we don't call it cap height ot x-height.

Reply:
@Spekular it's `-(top+bottom)=-2(center)`.

Reply:
@PhysSong @Spekular We think about space for note label so maybe change from `font` to `label` `noteLabelHeight`, `labelHeight`, `labelArea` or `labelSpace`?

Reply:
For making the meaning clear, I think we need something like `twoTimesBaselineYOffset`, but I think it's too long. We can use some other name and add a comment which describes what the value means.